src/components/Grid/Grid.test.jsx

```diff
@@ -26,7 +27,7 @@ describe.skip("Grid component", () => {
   });
 });
 
-xdescribe("Grid should populate assets and connections", () => {
+describe.skip("Grid should populate assets and connections", () => {
```

Comment:
```suggestion
describe("Grid should populate assets and connections", () => {
```
If intentional - add comment

Reply:
It is intentional, but was there before so I don't know the reason behind it